js/webviews.js

```diff
@@ -430,7 +435,7 @@ ipc.on('view-ipc', function (e, data) {
 
 setInterval(function () {
   captureCurrentTab()
-}, 30000)
+}, 3000)
```

Comment:
Ops... I forgot that. Will fix it shortly, but in the meantime, what is it used for?

Reply:
Because we can't draw anything directly on top of browserviews, there's no way to show the page in the background while the searchbar is open (or during the task overlay transition). To work around this, we capture a low-resolution image of the page. When the searchbar is opened, the view gets hidden, and the image is shown instead. We normally make a new image every time a new page is loaded, but in some situations (such as single-page apps, or if you scroll to a different part of the page) it's possible for the content of the page to change without the URL changing. This code makes a new image at 30-second intervals, so that the image is always at least somewhat close to what the actual page looks like, even if the URL hasn't changed in a while.

(I should probably add a comment explaining this to webviews.js; I'll probably do that after this PR is merged).

Reply:
Oh.. so it's doing a screenshot. I had interpretated "capture" as "grab keyboard and mouse" and was wondering why it was here.